algorithms/scaling/scaler.py

```diff
@@ -212,7 +212,7 @@ def perform_error_optimisation(self, update_Ih=True):
                 self.params.weighting.error_model, Ih_table
             )
         except (ValueError, RuntimeError) as e:
-            logger.info(e, exc_info=True)
+            logger.debug(e, exc_info=True)
```

Comment:
I think it would be best to keep the error message to info (without trace), so that the user knows that error modelling was unsuccessful for whatever reason/condition (there are a number in error modelling where ValueError is raised), so how about logger.info(e) followed by logger.debug(e, exc_info=True) ?

Reply:
I don't hold a strong opinion & one line is fairly harmless, so done in aa00a0d10b2144142c02f654c83b0983422015f5